src/index_emails.py

```diff
@@ -135,17 +187,21 @@ def convert_msg_to_json(msg):
     # Bodies...
     if tornado.options.options.index_bodies:
         result['body'] = ''
-        if msg.is_multipart():
-            for mpart in msg.get_payload():
-                if mpart is not None:
-                    mpart_payload = mpart.get_payload(decode=True)
-                    if mpart_payload is not None:
-                        result['body'] += strip_html_css_js(mpart_payload)
-        else:
-            result['body'] = strip_html_css_js(msg.get_payload(decode=True))
+        result['body_ignored_content_types'] = ''
+        result['body_filenames'] = ''
+
+        process_msg_body(msg,result)
 
+        # log body size
         result['body_size'] = len(result['body'])
 
+    # tags?
+    if tornado.options.options.tags:
+        kv_pairs = tornado.options.options.tags.split(",")
+        for kv_pair in kv_pairs:
+            kv = kv_pair.split("=")
```

Comment:
`kv` needs a `len(kv) == 2` check before it can be used.

Reply:
done